src/pages/AppList.svelte

```diff
@@ -101,7 +106,7 @@
                 />
               {/if}
             </div>
-            <div class="mzp-c-card-content">
+            <div class="mzp-c-card-content" style="pointer-events:none">
```

Comment:
Please add a comment here explaining why we need this and reference the bug in which this might be fixed.

Reply:
Done